packages/react-sdk/docusaurus/docs/React/03-ui-components/core/participant-view.mdx

```diff
@@ -3,8 +3,130 @@ id: participant-view
 title: Participant
 ---
 
-:::warning
+The `ParticipantView` component is a core component that renders a participant's video and plays the participant's audio.
 
-TODO: write about: https://github.com/GetStream/stream-video-js/tree/main/packages/react-sdk/src/core/components/ParticipantView
+It can toggle between video and avatar based on the participant's video state. It displays information about the participant, their device settings and connection quality. It also provides action buttons (for example, to pin the participant).
+
+To optimize performance,
+
+- the component won't stream video for participants that are not visible in the viewport
```

Comment:
*only when viewport element is set through `call.viewportTracker.setViewport`

We could probably direct them [here](https://github.com/GetStream/stream-video-js/blob/db0a25d1ac584d67e5b38d5cc4b3a0c834012991/packages/react-sdk/docusaurus/docs/React/06-ui-cookbook/13-custom-call-layout.mdx?plain=1#L314) for clarification.

Reply:
I understand the comment, but I've looked at the linked section, and if we link it here it looks like as if integratiors always have to deal with this, which is confusing. They only have to deal with that when they want a custom layout. So I think it's better not to mention that here, they don't have to know about this if they are using a defaul layout, if they do a custom one, they will see that part anyway.

Reply:
That's a good point. 👍